### ISSUE TITLE
Doesn't work with DrUms updated OS

### ISSUE THREAD

**Issue by StupidHoroscope** [StupidHoroscope/fks-eduke32#1]
Apparently missing libvorbisfile.so

**Comment by DrUm78** [comment]
Yeah I removed libvorbis package to fix VVVVVV game. You would then need to add this lib in your OPK I guess.

**Comment by DrUm78** [comment]
I fixed it by creating a modified OPK with the required libvorbis files and the following shell script:
```#!/bin/sh
# Detect whether libvorbis libs there already here then copy them if not
if [ ! -f /usr/lib/libvorbis.so.0 ]; then
	rw
	cp -f libvorbis.so.0 /usr/lib
	cp -f libvorbisfile.so.3 /usr/lib
	./eduke32 "$1"
	rm -f /usr/lib/libvorbis.so.0
	rm -f /usr/lib/libvorbisfile.so.3
	ro
else
	./eduke32 "$1"
fi```

That way, both VVVVVV and eDuke32 work with my custom OS.

**Comment by StupidHoroscope** [comment]
Fixed in v1.1
